omod/src/main/webapp/resources/jsonforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.json

```diff
@@ -1329,6 +1329,63 @@
                 "hideWhenExpression": "isEmpty(dsdmApproach) || dsdmApproach == '733144c0-c321-11e8-a355-529269fb1459' || dsdmApproach == '73313c96-c321-11e8-a355-529269fb1459' || dsdmApproach == '73313f20-c321-11e8-a355-529269fb1459' || dsdmApproach == '733139e4-c321-11e8-a355-529269fb1459'"
               }
             },
+            {
+              "label": "Pharmacy",
+              "type": "obs",
+              "required": false,
+              "id": "pharmacyDatasource",
+              "questionOptions": {
+                "rendering": "ui-select-extended",
+                "concept": "7a22cfcb-a272-4eff-968c-5e9467125a7b",
+                "datasource": {
```

Comment:
@akileng56 are you targeting cohorts or cohort types as custom data sources.

Reply:
@slubwama There is a dependant PR that needs to be merged first in the esm-ugandaemr-core - https://github.com/METS-Programme/esm-ugandaemr-core/pull/257

Reply:
@akileng56  did you mean https://github.com/METS-Programme/esm-ugandaemr-core/pull/262